src/lib/Log.ts

```diff
@@ -24,7 +24,7 @@ const Log = {
   error: (...args: unknown[]) => loggerInstance.error(...args),
 }
 
-const setLogConfiguration = (config: Omit<configLoggerType, 'levels'>) => {
+const setLogConfiguration = (config: Omit<configLoggerType<typeof consoleTransport, string>, 'levels'>) => {
```

Comment:
Dit werkt ook en is misschien wat netter:
```suggestion
const setLogConfiguration = (
  config: configLoggerType<transportFunctionType<object>, 'error' | 'warn' | 'info' | 'debug' | 'trace'>,
) => {